lang/es_ES.UTF-8/LC_MESSAGES/help.po

```diff
@@ -4313,3 +4313,13 @@ msgid "If checked, it won't  use the proxy configured in "
 "'Administration > Parameters > Centreon UI' to connect to the Remote Server."
 msgstr ""
 ""
+
+#: /www/include/Administration/parameters/ldap/help.po
+msgid "The interval, in hours, between two synchronizations of LDAP groups with Centreon ACLs. By default 1 hour."
+msgstr ""
```

Comment:
I'm scared that it displays nothing with ES translation
(cause msgid will be replaced by msgstr)